### PR TITLE
Update py-nexusforge

### PR DESCRIPTION
**Pull request by nabilalibou**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-prettytable/package.py [hunk 1]
```diff
@@ -12,9 +12,13 @@ class PyPrettytable(PythonPackage):
     appealing ASCII tables.
     """
 
-    homepage = "https://code.google.com/archive/p/prettytable/"
-    pypi = "prettytable/prettytable-0.7.2.tar.gz"
+    homepage = "https://github.com/jazzband/prettytable"
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-prettytable/package.py]: This should be done with `spack augment` to generate a package in `repo-patches`. This `builtin` repository will be reset to upstream's default on any conflict.

File: var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    homepage = "https://github.com/elastic/elasticsearch-dsl-py"
+    pypi = "elasticsearch-dsl/elasticsearch-dsl-7.4.0.tar.gz"
+
+    version('7.4.0', sha256='c4a7b93882918a413b63bed54018a1685d7410ffd8facbc860ee7fd57f214a6d')
+
+    depends_on('py-setuptools', type='build')
+    depends_on('py-six', type=('build', 'run'))
+    depends_on('py-python-dateutil', type=('build', 'run'))
+    depends_on('py-elasticsearch@7.0.0:7.999', type=('build', 'run'))
```
Comment (matz-e) [inline var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py:23]: Should also be in `repo-patches`.

**Review by matz-e — COMMENTED** [review]
Can you write to this repository? Please let me know if not. I think the CI only runs from PRs within the same repository.

File: var/spack/repos/builtin/packages/py-prettytable/package.py (hunk 1 above)
Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-prettytable/package.py]: Ok i see, i found the corresponding doc and i will follow it.

File: var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py (hunk 2 above)
Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py:23]: Ok, I'll do

File: var/spack/repos/builtin/packages/py-prettytable/package.py (hunk 1 above)
Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-prettytable/package.py]: > Can you write to this repository? Please let me know if not. I think the CI only runs from PRs within the same repository.

You mean if I can write in the repo-patches repo ?

Comment (matz-e) [reply var/spack/repos/builtin/packages/py-prettytable/package.py]: I meant if you can push a branch to this repository. For security reasons, it's simpler to restrict who can write to this repository and only execute CI for branches within the BlueBrain organization.

Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-prettytable/package.py]: Yes i tried and it seems that I can push commits into this repository

Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-prettytable/package.py]: Just seen that the last commit did not contained the good changes, I redo this quickly

File: var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py (hunk 2 above)
Comment (matz-e) [reply var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py:23]: Sorry, by "also" I meant that this file should only be in `repo-patches`, not here.

**Review by matz-e — COMMENTED** [review]
This still needs to be filed from a branch in `BlueBrain/spack`, not your own fork. Otherwise the CI won't run.

**Comment by nabilalibou** [comment]
> This still needs to be filed from a branch in `BlueBrain/spack`, not your own fork. Otherwise the CI won't run.

Aaah ok sorry i did not understood that at first. Ok I'll try creating a new branch in this repo

File: var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py (hunk 2 above)
Comment (nabilalibou) [reply var/spack/repos/builtin/packages/py-elasticsearch-dsl/package.py:23]: It is me, i forgot to remove it

**Comment by matz-e** [comment]
> > This still needs to be filed from a branch in `BlueBrain/spack`, not your own fork. Otherwise the CI won't run.
> 
> Aaah ok sorry i did not understood that at first. Ok I'll try creating a new branch in this repo

Thanks! I guess I should make this more explicit in the docs!

**Pull request closed without merging**